operator/controllers/internal/csiscaleoperator/csiscaleoperator_package.go

```diff
@@ -0,0 +1,667 @@
+package csiscaleoperator
+
+import (
+	securityv1 "github.com/openshift/api/security/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "github.com/IBM/ibm-spectrum-scale-csi/api/v1"
+	"github.com/IBM/ibm-spectrum-scale-csi/controllers/config"
+	"github.com/IBM/ibm-spectrum-scale-csi/controllers/util/boolptr"
```

Comment:
Please change this to :

       v1 "github.com/IBM/ibm-spectrum-scale-csi/operator/api/v1"
	"github.com/IBM/ibm-spectrum-scale-csi/operator/controllers/config"
	"github.com/IBM/ibm-spectrum-scale-csi/operator/controllers/util/boolptr"

Reply:
Done